src/main/java/chessmaster/parser/Parser.java

```diff
@@ -91,6 +91,38 @@ public static Move parseMove(String in, ChessBoard board) throws ParseCoordinate
         return new Move(from, to, relevantPiece);
     }
 
+    /**
+     * Parses a chess move from user input and creates a Move object. Used to read
+     * user inputs from the file while setting up the chess game.
+     *
+     * @param in    The file input string with 2 algebraic coordinate notations
+     *              (e.g., "e2 e4").
+     * @param board The ChessBoard where the move is taking place.
+     * @return Move object containing information about the move to be made.
+     *
+     * @throws ParseCoordinateException   If the string entered is not in the
+     *                                    algebraic coordinate notation.
+     * @throws NullPieceException         If the tile at the coordinate is empty
+     */
+    public static Move parseMoveFile(String in, ChessBoard board) throws ParseCoordinateException,
+            NullPieceException, MoveOpponentPieceException {
+
+        String[] parseArray = in.split("\\s+", 2);
+        if (parseArray.length < 2) {
+            throw new ParseCoordinateException();
+        }
+
+        Coordinate from = Coordinate.parseAlgebraicCoor(parseArray[0]);
+        Coordinate to = Coordinate.parseAlgebraicCoor(parseArray[1]);
+
+        ChessPiece relevantPiece = board.getPieceAtCoor(from);
+        if (relevantPiece.isEmptyPiece()) {
+            throw new NullPieceException(from);
+        }
+
+        return new Move(from, to, relevantPiece);
+    }
+
```

Comment:
Is this the same as `parseMove` without the `MoveOpponentPieceException` check?
If so, we can just use the normal parseMove

Reply:
The moveopponentpiece exception check is important to remove, otherwise its unable to execute the CPU moves